client/src/components/Onboarding/OnboardingStatus.module.scss

```diff
@@ -3,11 +3,6 @@
   margin-right: 0.5em;
 }
 
-.loading-icon {
-  display: inherit;
-  width: inherit;
-}
-
```

Comment:
This class was not applied (would have been done via `styleName`). I had tried to apply it in a previous commit, but I found replicating an existing similar instance of styling was a clearer solution that did not introduce an old bug.